spec/features/reservations_spec.rb

```diff
@@ -552,7 +552,6 @@
       select('10', from: 'items_per_page') # show more than 1 item (default)
```

Comment:
Fair point, although since it ultimately takes the default value from the app configs I feel like we should update `FeatureHelpers#app_setup` to use `mock_app_config` instead (passing in the `attributes_for` to give it a fully functional object) and then we can just stub the relevant parameter in these tests. This has the benefit of skipping all the database hits when checking `AppConfig.first` throughout our code base but also allows us to easily stub out configs in feature specs whenever we want. Additionally, in these specs, it means we skip an entire request which is pretty huge, time-wise. Does that make sense?